src/templates/uploads.html

```diff
@@ -27,6 +48,18 @@ <h3 title='{{ 'Toggle visibility'|trans }}' data-action='toggle-next' data-icon=
                         <i class='fas fa-fw fa-image mr-1'></i>{{ 'Insert in text at cursor position'|trans }}</a>
                     {% endif %}
 
+                    <!-- INSERT IN TEXT in edit mode for plain text -->
+                    {% if ext matches '/txt$/i' %}
+                      <a class='dropdown-item' data-action='insert-plain-text' data-storage='{{ upload.storage }}' data-path='{{ upload.long_name }}'>
+                        <i class='fas fa-fw fa-file-import mr-1'></i>{{ 'Insert in text at cursor position'|trans }}</a>
+                    {% endif %}
+
+                    <!-- Show content of plain text -->
+                    {% if ext matches '/txt$/i' %}
```

Comment:
I guess you didn't merge this IF with the one above because of the probable extension to other file types.